.github/workflows/stale-issue-mgr.yaml

```diff
@@ -0,0 +1,31 @@
+# Stale issue handler (could also be extended for stale PRs in future)
+
+# This action will: 
+# * Add a label "stale" on issues after 60 days of inactivity and comment on them if they also have the "external" label
+# * Close the stale issues and pull requests after 7 days of inactivity
+# * If an update/comment occur on stale issues or pull requests, the stale label will be removed and the timer will restart if they have the "external" label
+
+# Runs every morning at 1AM: We don't want people having to wait a whole week to get the stale label removed 
+# Docs: https://github.com/actions/stale
+
+name: 'Label and close stale issues'
+on:
+  schedule:
+    - cron: '0 1 * * *'
```

Comment:
I changed the execution to every day, to have more frequent un-stales if people comment, instead of waiting a week and pinging us asking to remove the label.